### ISSUE TITLE
Angular 15 compiler-cli TypeScript  4.9.3 support

### ISSUE THREAD

**Issue by himyjan** [angular/angular#48097]
### Which @angular/* package(s) are the source of the bug?

compiler-cli

### Is this a regression?

No

### Description

The Angular Compiler requires TypeScript >=4.8.2 and <4.9.0 but 4.9.3 was found instead.

```
node_modules\@angular\compiler-cli\bundles\chunk-KGXXQYFO.js:268:11
  Code:
      267 |   if (compareVersions(version, minVersion) < 0 || compareVersions(version, maxVersion) >= 0) {
    > 268 |     throw new Error(`The Angular Compiler requires TypeScript >=${minVersion} and <${maxVersion} but ${version} was found instead.`);
          |           ^
      269 |   }
      270 | }
      271 | function verifySupportedTypeScriptVersion() {
```

**Comment by hrstmr** [comment]
Would be also nice if we can have support for 4.9.3 in older versions of angular as well, at least v14 if not v13.
🙂

**Comment by JoostK** [comment]
TypeScript 4.9 was released too late to be included in 15.0, so it's not yet supported. It will become available in 15.1.
